engine/src/io/DataLoader.cpp

```diff
@@ -214,10 +214,9 @@ std::unique_ptr<ral::frame::BlazingTable> data_loader::load_batch(
 	Context * context,
 	const std::vector<size_t> & column_indices_in,
 	const Schema & schema,
-	std::string user_readable_file_handle,
 	data_handle file_data_handle,
 	size_t file_index,
-	size_t batch_id) {
+	cudf::size_type row_group_id) {
```

Comment:
why are we changing from batch id to row_group_id? different file formats use different words im more curious than asking for a change here

Reply:
I agree. In the logic of the algorithm, those partitions are considered as batches, but when reading the files they end up being called row groups (i.e. for parquet files), maybe this change was made to maintain some uniform terminology.